### ISSUE TITLE
SRSUE stuck with attaching UE

### ISSUE THREAD

**Issue by iamniki01** [srsran/srsRAN_Project#721]
## Issue Description ##
I'm following the recent tutorial 'srsRAN gNB with srsUE' also using DPDK to avoid packet loss from here https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html. The UE is not attaching after many tries and I have no clue where I am going wrong, and when I check the logs of UE (attached as an UE.log) got CRC Failed warning and loops via it forever.

## Setup Details ##
Using 2 USRP of x410 as gNB AND SRSue with high compatible machines AMD with 16 cores

## Expected Behavior ##
srsue connecting to the gNB.

## Actual Behaviour ##
![image](https://github.com/srsran/srsRAN_Project/assets/67425228/bc5ae71a-8e25-4fd0-973e-81043133af63)
![image](https://github.com/srsran/srsRAN_Project/assets/67425228/8f5b0417-2e45-44ce-80e1-7e40dd9597a8)

## Steps to reproduce the problem ##
Here I attached the config files of both gnb and Ue. Also its Logs and its pcaps.

## Additional Information ##
NOTE: I created the issue at srsran_4g. then I noticed people are more active in this repository so I am moving this here.
https://github.com/srsran/srsRAN_4G/issues/1358, one can obsolete either of this. 

Any insights and assistance will be appreciated. I am looking forward for your quick response.

P.S: Using both srsran and srsue with latest release (and updates)
[Logs_Pcaps.zip](https://github.com/user-attachments/files/16158698/Logs_Pcaps.zip)

**Comment by pgawlowicz** [comment]
could you try to add the following parameters to the srsue:
```
[expert]
lte_sample_rates = true

```

see here: https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html#usrp-x300-x310

**Comment by iamniki01** [comment]
> could you try to add the following parameters to the srsue:
> 
> ```
> [expert]
> lte_sample_rates = true
> ```
> 
> see here: https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html#usrp-x300-x310

Hi Thanks, It passed the stage atleast, thanks for an eye opener. However, it is not connecting to the AMF and not getting its IP too. 
1. UE connectivity 
![image](https://github.com/user-attachments/assets/c6d51a36-77b1-4985-84f7-388ff182c015)
 
2. This amf logs shows that only about gnb, ue is not connecting.
![image](https://github.com/user-attachments/assets/bbb71165-7341-42c4-9c3d-9d83cf256bcb)

**Comment by pgawlowicz** [comment]
Could you put gnb logs and ngap pcap?

**Comment by iamniki01** [comment]
> Could you put gnb logs and ngap pcap?

That was quick, Thanks:) Here is the complete logs and pcap of both gnb and ue.
[logs_pcaps.zip](https://github.com/user-attachments/files/16189871/logs_pcaps.zip)

**Comment by pgawlowicz** [comment]
could you post the gnb and srsue logs?

**Comment by iamniki01** [comment]
> could you post the gnb and srsue logs?

Hi again,

I played around with `freq_offset` and `tx & Rx gain`, and it worked. I started receiving the IP and can now see the traffic between the gNB and UE without network breaks. However, if I run the UE five times, it might fail once with an RRC release after a successful connection of about five seconds. I realize that the cause of the break is if the UE gets the `Late` status. If you know how to solve this minor issue, that would be helpful; if not, this issue can be closed now. Thanks for the great help!
